### ISSUE TITLE
The question about the result of scratch-Pub of MOTA=57.4

### ISSUE THREAD

**Issue by TTTREE** [xingyizhou/CenterTrack#202]
Thanks for your great work!
I am a student and doing some experiments on CenterTrack. I can get the correct results of MOTA=63.1(public) and MOTA=66.1(private). 
But I can't get the MOTA=57.4 of scratch-Pub in Table 9.
I am using this script, loading the model named " mot17_half_sc.pth "
python test.py tracking --exp_id mot17_half_sc --dataset mot --dataset_version 17halfval --pre_hm --ltrb_amodal --track_thresh 0.4 --pre_thresh 0.5 --load_model ../mot17_half_sc.pth
1. Am I using the correct script to get the result of MOTA=57.4? Could you please help me with this?

2. When I do some tests, I find the track_thresh is essential. In this paper, track_thresh is set 0.4 according to Table 11. But I find track_thresh=0.3 or track_thresh=0.2 can get the higher result of MOTA, why do you choose track_thresh=0.4 other than 0.2 or 0.3? 

3.How can I choose the proper track_thresh on my own trained model? Do I have to set the same track_thresh when I compare my method to CenterTrack, which my method modify based on CenterTrack?

Thank you very much!

**Comment by xingyizhou** [comment]
Hi,
  Thank you for your questions.

  1. It should be correct as far as I can see. What is the number you get from that? I didn't carefully check the numbers in the supplement table before code release. The paper numbers might be from another model. What you get from the code should be correct.
  2. I observed this too as is ablated in supplement Table. 11 I use 0.4 as swept in Table 11.
  3. You should probably search for the optimal threshold for your data on your validation set ...

**Comment by TTTREE** [comment]
I get MOTA=58.0 .
Thanks for your kind reply!!
